static/scss/includes/variables.scss

```diff
@@ -17,6 +17,8 @@ $color-mpp-gray-30: #ECECEC;
 
 $box-shadow-mpp-sm: 0px 0px 4px rgba(0, 0, 0, .12);
 
+// Custom width
+$content-sm-md: 546px;
```

Comment:
_Goldilocks_ width in between Protocol small and medium sizes.

Reply:
TODO: Consider proposing this on Protocol?

Reply:
Could you somehow `calc($content-sm - $content-xs)/2` ? That's 560px. 👀

Reply:
@codemist Great idea. Dropping that pesky in-between variable.